r/pkgdown/_pkgdown.yml

```
@@ -74,6 +74,7 @@ reference:
   - call_peaks_tile
   - merge_peaks_iterative
   - write_insertion_bedgraph
+  - call_macs_peaks
```

```suggestion
  - call_macs_peaks
  - call_peaks_tile
  - merge_peaks_iterative
  - write_insertion_bedgraph
```
Re-order `call_macs_peaks` to slot in next to the other peak-calling oriented function (but coming first since it's a lot better)